src/test/rustdoc/default_trait_method.rs

```diff
@@ -0,0 +1,14 @@
+#![feature(specialization)]
+
+pub trait Item {
+    fn foo();
+    fn bar();
+}
+
+// @has default_trait_method/trait.Item.html
+// @has - '//*[@id="method.foo"]' 'default fn foo()'
+// @has - '//*[@id="method.bar"]' 'fn bar()'
```

Comment:
Is there a way we can make sure that `bar()` does not have the `default` keyword? Would `@count - 'default' 1` work for this test?

Reply:
Good question. I'll check.

Reply:
`@count` is used to count DOM elements, not strings occurrence that won't work. I strengthen the test.